src/index.js

```diff
@@ -19,8 +19,11 @@ app.use(morgan('dev'));
 app.use((req, res, next) => {
   if (req.method === 'OPTIONS') {
     res.header('Access-Control-Allow-Origin', '*');
-    res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE,OPTIONS');
-    res.header('Access-Control-Allow-Headers', 'Content-Type, Authorization, Content-Length, X-Requested-With, X-Redmine-API-Key');
+    res.header('Access-Control-Allow-Methods', 'GET, PUT, POST, DELETE, OPTIONS, PATCH');
```

Comment:
👍 nice
just as a minor, what about to introduce `config.json`, where you can configure options. for example, headers?

Reply:
yeah I was thinking about it, but this was just quick hotfix